docs/src/addingops.md

```diff
@@ -0,0 +1,38 @@
+TensorFlow has a huge number of operations.
+TensorFlow.jl implements many of them; but their are many more still to be added.
+Linking up to the TensorFlow API to implement a new operation is a fun and easy way to contribute.
+
+See also the [TensorFlow guide for writing language bindings.](https://www.tensorflow.org/extend/language_bindings).
+
+
+Once you have identified the operation you want to add, the following steps should be taken.
+(If you like TDD, feel free to bring the tests first)
+
+ 1. Implement the opteration
+     - This normally means adding it to one of the files `/src/ops/`
+     - You can find the operation's "true name", and its input and attrinbute definitions by looking in the [TensorFlow ops list](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/ops/ops.pbtxt)
+     - You can base the implementation on something already there
+     - Don't forget to shift indexs to be 1 based, as tensorflow defaults to zero based.\
```

Comment:
Operation here is also spelt wrong.
I should edit this in something with spellcheck.